lib/modules/manager/tekton/__snapshots__/extract.spec.ts.snap

```diff
@@ -0,0 +1,133 @@
+      "depType": "tekton-bundle",
+      "replaceString": "gcr.io/tekton-releases/catalog/upstream/trigger-template-task-run-resolver",
+    },
+    {
+      "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
+      "currentDigest": undefined,
+      "currentValue": undefined,
+      "datasource": "docker",
+      "depName": "gcr.io/tekton-releases/catalog/upstream/trigger-template-pipeline-run",
+      "depType": "tekton-bundle",
+      "replaceString": "gcr.io/tekton-releases/catalog/upstream/trigger-template-pipeline-run",
+    },
+    {
+      "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
+      "currentDigest": undefined,
+      "currentValue": undefined,
+      "datasource": "docker",
+      "depName": "gcr.io/tekton-releases/catalog/upstream/trigger-template-pipeline-run-resolver",
+      "depType": "tekton-bundle",
+      "replaceString": "gcr.io/tekton-releases/catalog/upstream/trigger-template-pipeline-run-resolver",
+    },
+    {
+      "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
+      "currentDigest": undefined,
+      "currentValue": undefined,
+      "datasource": "docker",
+      "depName": "gcr.io/tekton-releases/catalog/upstream/task-run",
+      "depType": "tekton-bundle",
+      "replaceString": "gcr.io/tekton-releases/catalog/upstream/task-run",
+    },
+    {
+      "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
+      "currentDigest": undefined,
+      "currentValue": undefined,
+      "datasource": "docker",
+      "depName": "gcr.io/tekton-releases/catalog/upstream/task-run-resolver",
+      "depType": "tekton-bundle",
+      "replaceString": "gcr.io/tekton-releases/catalog/upstream/task-run-resolver",
+    },
+    {
+      "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
+      "currentDigest": undefined,
+      "currentValue": undefined,
+      "datasource": "docker",
+      "depName": "gcr.io/tekton-releases/catalog/upstream/pipeline-run",
+      "depType": "tekton-bundle",
+      "replaceString": "gcr.io/tekton-releases/catalog/upstream/pipeline-run",
+    },
+    {
+      "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
+      "currentDigest": undefined,
+      "currentValue": undefined,
+      "datasource": "docker",
+      "depName": "gcr.io/tekton-releases/catalog/upstream/pipeline-run-resolver",
+      "depType": "tekton-bundle",
+      "replaceString": "gcr.io/tekton-releases/catalog/upstream/pipeline-run-resolver",
+    },
+    {
+      "depType": "tekton-bundle",
+      "skipReason": "invalid-value",
+    },
+    {
+      "depType": "tekton-bundle",
+      "skipReason": "invalid-value",
+    },
```

Comment:
should they've been filtered because they have no depName?

Reply:
No. This is a side effect of implementing the suggested [change](https://github.com/renovatebot/renovate/pull/17289#discussion_r955313352), see discussion there.